mypy-baseline.txt

```diff
@@ -874,3 +860,16 @@ posthog/api/test/batch_exports/test_update.py:0: error: Value of type "BatchExpo
 posthog/api/test/batch_exports/test_update.py:0: error: Value of type "BatchExport" is not indexable  [index]
 posthog/api/test/batch_exports/test_update.py:0: error: Value of type "BatchExport" is not indexable  [index]
 posthog/api/test/batch_exports/test_pause.py:0: error: "batch_export_delete_schedule" does not return a value (it only ever returns None)  [func-returns-value]
+posthog/api/query.py:0: error: Statement is unreachable  [unreachable]
+posthog/api/test/test_capture.py:0: error: Statement is unreachable  [unreachable]
+posthog/api/test/test_capture.py:0: error: Incompatible return value type (got "_MonkeyPatchedWSGIResponse", expected "HttpResponse")  [return-value]
+posthog/api/test/test_capture.py:0: error: Module has no attribute "utc"  [attr-defined]
+posthog/api/test/test_capture.py:0: error: Unpacked dict entry 0 has incompatible type "Collection[str]"; expected "SupportsKeysAndGetItem[str, dict[Never, Never]]"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Unpacked dict entry 0 has incompatible type "Collection[str]"; expected "SupportsKeysAndGetItem[str, dict[Never, Never]]"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Unpacked dict entry 0 has incompatible type "Collection[str]"; expected "SupportsKeysAndGetItem[str, dict[Never, Never]]"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Dict entry 0 has incompatible type "str": "float"; expected "str": "int"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Dict entry 0 has incompatible type "str": "float"; expected "str": "int"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Dict entry 0 has incompatible type "str": "float"; expected "str": "int"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Dict entry 0 has incompatible type "str": "float"; expected "str": "int"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Dict entry 0 has incompatible type "str": "float"; expected "str": "int"  [dict-item]
+posthog/api/test/test_capture.py:0: error: Dict entry 0 has incompatible type "str": "float"; expected "str": "int"  [dict-item]
```

Comment:
By some reason, the baseline file is sorted in a different order on my machine. I fixed all errors appeared after the Pydantic version bump.